src/main/workspace/tree/visitor/CollectFilesVisitor.ts

```
@@ -0,0 +1,27 @@
+    if (this.banned && this.banned.evaluate(folder)) return;
+    const children = folder.getChildren();
+    for (let i = 0; i < children.length; i += 1) {
+      children[i].accept<void>(this);
+    }
+  }
+
+  VisitFile(file: File): void {
+    if (this.banned && this.banned.evaluate(file)) return;
+    this.files.push(file);
+  }
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)